docs/snippets/AUTO1.cpp

```diff
@@ -0,0 +1,13 @@
+#include <ie_core.hpp>
+
+int main() {
+//! [part1]
+    InferenceEngine::Core ie;
+    InferenceEngine::CNNNetwork network = ie.ReadNetwork("sample.xml");
+    //NEW IE-CENTRIC API, the "AUTO" plugin is (globally) pre-configured with the explicit option:
```

Comment:
I think I have added a change that explains that the 2 LoadNEtworks are equivalent and while first does rely on the pre-configuring the AUTO (see prev comment) the second does the conf for the specific ExecutableNetworkm via passing the device list to the LoadNetwork .

Notice that third option  is ie.LoadNetwork(network, "AUTO:CPU,GPU");

Reply:
Got it, three options for ``ie.LoadNetwork`` there now;